tests/tpch/test_dask.py

```diff
@@ -306,3 +306,78 @@ def test_query_7(client, dataset_path, fs):
         by=["supp_nation", "cust_nation", "l_year"],
         ascending=True,
     ).compute()
+
+
+@pytest.mark.shuffle_p2p
+def test_query_10(client, dataset_path, fs):
+    """
+    select
+        c_custkey,
+        c_name,
+        round(sum(l_extendedprice * (1 - l_discount)), 2) as revenue,
+        c_acctbal,
+        n_name,
+        c_address,
+        c_phone,
+        c_comment
+    from
+        customer,
+        orders,
+        lineitem,
+        nation
+    where
+        c_custkey = o_custkey
+        and l_orderkey = o_orderkey
+        and o_orderdate >= date '1993-10-01'
+        and o_orderdate < date '1993-10-01' + interval '3' month
+        and l_returnflag = 'R'
+        and c_nationkey = n_nationkey
+    group by
+        c_custkey,
+        c_name,
+        c_acctbal,
+        c_phone,
+        n_name,
+        c_address,
+        c_comment
+    order by
+        revenue desc
+    limit 20
+    """
+    customer = dd.read_parquet(dataset_path + "customer", filesystem=fs)
+    orders = dd.read_parquet(dataset_path + "orders", filesystem=fs)
+    lineitem = dd.read_parquet(dataset_path + "lineitem", filesystem=fs)
+    nation = dd.read_parquet(dataset_path + "nation", filesystem=fs)
+
+    orderdate_from = datetime.strptime("1993-10-01", "%Y-%m-%d")
+    orderdate_to = orderdate_from + timedelta(days=3 * (365 / 12))
```

Comment:
The math doesn't check out here, the date is off by 18 hours (`1993-12-31 06:00:00`).

Instead of dealing with datetime arithmetics here, I'd just hard-code it. This is what has already been done for existing queries.
```suggestion
    orderdate_to = datetime.strptime("1994-01-01", "%Y-%m-%d")
```